lib/avo/fields/file_field.rb

```diff
@@ -24,6 +24,16 @@ def initialize(id, **args, &block)
       def path
         rails_blob_url value
       end
+
+      def value
+        final_value = super
+        # On edit view always show the persisted image. Related: issue#3008
+        if final_value.instance_of?(ActiveStorage::Attached::One) && view == "edit"
+          persisted_record = record.class.find_by(id: record.id)
+          final_value = persisted_record.send(@for_attribute || id)
```

Comment:
@binarygit do you think it's possible to use some active record helper to reload `@for_attribute || id` from the `@record` while keeping other attributes dirty? This way we can avoid the manual `find_by` query...

Reply:
I’ll look into it paul

Reply:
@Paul-Bob I looked but there's no straight forward way to do this. I tried to dup the record and clear specific changes but the changes propagated to the original record too which we shouldn't let happen.